README.md

```diff
@@ -5,10 +5,34 @@ Cross validation of machine-learning models on Faculty platform. At present, the
 ```python
 faculty_xval.validation.JobsCrossValidator
 ```
-Additional information is found in the notebooks of the `examples` directory.
+Additional information is found in the example notebooks provided. Please have a look at the section `Try out the examples` below.
 
 The package supports `keras` and `sklearn` models. Whilst one can write custom models that are compatible with `faculty-xval`, no guarantee is given that the package handles these situations correctly, in particular because of issues concerning the randomisation of weights.
 
+Two sets of installation instructions are provided below:
+* If you would like to simply use `faculty-xval`, please follow the `User installation instructions`.
+* If you would like to develop `faculty-xval` further, please follow the `Developer installation instructions`.
+
+## User installation instructions
+
+##### Create an environment
+In your project on Faculty platform, create an environment named `faculty_xval`. In the `PYTHON` section, select `Python 3` and `pip` from the dropdown menus. Then, type `faculty-xval` in the text box, and click on the `ADD` button.
+
+The environment installs the package `faculty-xval`, and should be applied on every server that you create; this includes both 'normal' interactive servers and job servers, as explained next.
+
+##### Create a job definition
+Create a new job definition named `cross_validation`. In the `COMMAND` section, paste the following:
+
+`faculty_xval_jobs_xval $in_paths`
+
+Then, add a `PARAMETER` with the name `in_paths`, and ensure that the `Make field mandatory` box is checked.
+
+Finally, under `SERVER SETTINGS`, add `faculty_xval` to the `ENVIRONMENTS` section.
+
+For cross-validation jobs that are computationally intensive, we recommend using dedicated servers as opposed to running in the cluster. To achieve this, click on `Large and GPU servers` under `SERVER RESOURCES`, and select an appropriate server type from the dropdown menu.
```

Comment:
```suggestion
For cross-validation jobs that are computationally intensive, we recommend using dedicated servers as opposed to running on shared infrastructure. To achieve this, click on `Large and GPU servers` under `SERVER RESOURCES`, and select an appropriate server type from the dropdown menu.
```

Reply:
Looks good. I was lacking the correct terminology, here.